pkg/operator/workloads/metrics.go

```diff
@@ -244,11 +244,25 @@ func getAPIDimensions(appName string, api *context.API) []*cloudwatch.Dimension
 	}
 }
 
+func getAPIDimensionsCounter(appName string, api *context.API) []*cloudwatch.Dimension {
+	return append(getAPIDimensions(appName, api), &cloudwatch.Dimension{
```

Comment:
Nit: same comment about the append formatting, if you agree (and if gofmt agrees haha)